lib/logstash/filters/cidr.rb

```diff
@@ -52,15 +123,37 @@ def filter(event)
     end
     address.compact!
 
-    network = @network.collect do |n|
-      begin
-        IPAddr.new(event.sprintf(n))
-      rescue ArgumentError => e
-        @logger.warn("Invalid IP network, skipping", :network => n, :event => event)
-        nil
+    if @network_path #in case we are getting networks from a file
+      if needs_refresh?
+        lock_for_read do
+          if needs_refresh?
+      	    load_file
+      	    @next_refresh = Time.now() + @refresh_interval
+	  end
+        end
+      end
+
+      network = @network.collect do |n|
+        begin
+          IPAddr.new(n)
+  	    rescue ArgumentError => e
+              @logger.warn("Invalid IP network, skipping", :network => n, :event => event)
+  	    end
+        end
```

Comment:
The indentation of this block of code if off in various ways and not consistent with the similar code in the else block a few lines down. Suggestion:

```
network = @network.collect do |n|
  begin
    IPAddr.new(n)
  rescue ArgumentError => e
    @logger.warn("Invalid IP network, skipping", :network => n, :event => event)
  end
end
```